src/botPage/view/Dialogs/Limits.js

```diff
@@ -27,10 +28,23 @@ class LimitsContent extends PureComponent {
             }
         } else {
             this.setState({
-                error: translate('Both number of trades and loss amount have to be positive values.'),
+                error: translate('Please enter maximum number of trades and maximum loss amount'),
             });
         }
     }
+    componentDidMount() {
+        const cleanupLayout = () => {
+            $(this.maxTradesDiv).val('');
```

Comment:
Better to set the input values from a local state and set the local state to empty string `''` here.